util/test/convert_start_test_log_to_junit_xml.py

```diff
@@ -116,7 +158,7 @@ def _parse_start_test_log(start_test_log):
             test_start_skip, test_end_skip = _get_block(
                 sub_test_lines,
                 '[test: ',
-                '[Skipping')
+                SKIP_MARKERS)
```

Comment:
_get_block takes a single element right? not a list

Reply:
See above, tuples can be used as arguments to startswith